packages/launchpad/src/setup/AlphaLabel.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <span class="text-warning-600 bg-warning-50 border-warning-300 border-1 border-transparent rounded p-2px ml-4px pr-3px">
```

Comment:
Similarly, here we could have a `.visually-hidden` piece of text that clarifies the work "alpha" refers to Cypress support of the framework, since it is read aloud as something like "nuxt alpha". We could prefix "alpha" with "support is in " or something similar.